packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -29,7 +30,7 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 
 	return {
 		name: 'vite-plugin-cloudflare',
-		config() {
+		config(config) {
```

Comment:
This argument is no longer needed:

```suggestion
		config() {
```

Reply:
https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/57/commits/fc66dec4f8bc8f70c08a483cd0a5e21ba0dbab02